### DISCUSSION TITLE
can we get a video tutorial.

### DISCUSSION THREAD

**Discussion by catv1** [Genesis-Embodied-AI/genesis-world#256] [Ideas]
I have been tearing my own hair out for hours at this installation and other things, and the documents don't help.

**Comment by zhanglepy** [comment]
hello ， I am also very concerned about this issue

**Comment by zhouxian** [comment]
What exactly is the issue you are facing? In general pip install would work for most of the people

**Reply by Pratz1337** [reply]
Alot of the stuff is not working with Python 3.12.1 cause of open3d model is not compatible with it. If I try lowering the version of my Python to 3.9 or 3.11 still there are some issues with OpenGL library
